Things.md

```diff
@@ -0,0 +1,117 @@
+This is a general guide about Things for all FullScreenShenanigans projects. This guide will show you how in-game Things are stored and manipulated. Everything you see in the game (trees, blocks, the player, etc.) is a Thing.  
+The Thing class is subclassed by a new class for everything (Tree class, Block class, Player class, etc.). These Things can be manipulated to change their location and properties.  
+All the following code samples are written specific to `FSM` (FullScreenMario) but work for all FullScreenShenanigans projects (`FSM`, `FSP`, etc.).  
```

Comment:
Could be worded a little better: you do have to switch the `FSM` to work with other games.